src/mmrelay/meshtastic/health.py

```
@@ -0,0 +1,746 @@
+    "_missing_probe_wait_error",
+    "_missing_received_nak_error",
+    "_prune_health_probe_tracking",
+    "_probe_device_connection",
+    "_reset_probe_ack_state",
+    "_seed_connect_time_skew",
+    "_set_probe_ack_flag_from_packet",
+    "_track_health_probe_request_id",
+    "_wait_for_probe_ack",
+    "check_connection",
+    "requires_continuous_health_monitor",
+]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)